deploy/kubernetes/helm/che/values.yaml

```diff
@@ -32,6 +32,8 @@ global:
   #  This value can be passed if custom Oidc provider is used, and there is no need to deploy keycloak in multiuser mode
   #  default (if empty) is true
   #cheDedicatedKeycloak: false
+  ingressClass: "ingress"
+  #ingressClass: "addon-http-application-routing" ## This may fix the issue around bootstraping your stacks on Azure AKS.
```

Comment:
What is right value for `.global.ingressAnnotationsPrefix` if `.global.ingressClass` is set to `addon-http-application-routing`? Will default value `nginx.` will work fine?